Contents/mods/Perdition/media/scripts/atm_WeaponTrader_recipes.txt

```diff
@@ -0,0 +1,319 @@
+
+        }
+    recipe 7.62x39 ammo box
+        {
+        destroy Money=200,
+        keep WeaponTrader,
+
+        Result:	762x39Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe 7.62x51 ammo box
+        {
+        destroy Money=200,
+        keep WeaponTrader,
+
+        Result:	762x51Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .44 ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	Bullets44Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .44-40 ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	Bullets4440Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .38 ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	Bullets38Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .357 ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	Bullets357Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .223 ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	223Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .308 ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	308Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .22LR ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	Bullets22Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .30-06 ammo box
+        {
+        destroy Money=150,
+        keep WeaponTrader,
+
+        Result:	Bullets3006Box,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe .22 Silencer
+        {
+        destroy Money=3000,
+        keep WeaponTrader,
+
+        Result:	22Silencer,
+        CanBeDoneFromFloor:true,
+        Category:Money,
+        Time:5,
+        OnTest:Change_All_For_Money_OnTest
+
+        }
+    recipe 9mm Silencer
+        {
+        destroy Money=3000,
+        keep WeaponTrader,
+
+        Result:	9mmSilencer,
```

Comment:
Much like the w900 comment, you also need to do the same for the firearms mod if the module for these items is not `Base`